README.md

```diff
@@ -10,30 +13,41 @@ Join the community on the [ContinualAI slack](https://join.slack.com/t/continual
 - Install Unix command line tools (like `rm`, and `make`) 
     - On Linux, you already have everything
     - On Mac, install `homebrew` and through homebrew, the `make` package
-    - On Windows... idk, wipe your hard drive and install a proper OS? (Or try Cygwin or Windows Subsystem for Linux)
+    - On Windows, not 100% sure, but try using Powershell, Cygwin or Windows Subsystem for Linux
 - Install Miniconda/Anaconda and have the conda/python/pip binaries for that available on your `$PATH` and in your shell.
 
 
 ## Setup
 
-1. Get your API Authentication token from the [EvalAI Website in your profile](https://eval.ai/web/profile).
+1. Get your API Authentication token from [you profile on the EvalAI Website](https://eval.ai/web/profile),
+   and copy it into the `evalai_token.txt` file.
 
-2. From the root directory of the repository, run the following command:
+2. From the root directory of the repository, run the `./install.sh` script and activate the `clcomp21` environment.
 
 ```console
-./install.sh <EVALAI_TOKEN>
+./install.sh
 conda activate clcomp21
 ```
 
 ## Creating Your Solution
 
-Develop a new solution in the `submission/` folder. 
+Submissions should be contained in the `submission/` folder. You can draw inspiration from the following examples:
+- [submission/example_method.py](submission/example_method.py):
+    Standard neural net classifier without any CL-related mechanism. Works in the SL
+    track, but has very poor performance.
+
+- [submission/example_reg_method.py](submission/example_reg_method.py):
+    Adds a simple regularization loss to the example SL method above. Still exhibits
+    poor performance.
+
+- [submission/dummy_method.py](submission/dummy_method.py):
+    Model-free example that outputs random predictions/actions. Applicable to all tracks (RL and SL).
 
-You can draw inspiration from `submission/classification_method.py` which is a standard neural net classifier, which kinda works in the SL setting. The `submission/dummy_target.py` is a model-free example that outputs random predictions/actions but doesn't learn anything.
+- (More to be added shortly)
 
-Make sure the `submission/submission.py` script actually calls your method.
+Make sure to change the contents of `submission/submission.py`, so that the various `get_method` actually return your method to use for each task (`get_method_sl` -> SL track, `get_method_rl` -> RL Track, `get_method` -> Both/Bonus track)
 
-For more details on how to develop methods within our framework and to understand how to leverage its functionalities, algorithms, and models, please check out [Sequoia, the library that this is all based on](https://github.com/lebrice/Sequoia/).
+For more information on the various CL Settings and Methods available in Sequoia, please check out [the Sequoia repository](https://github.com/lebrice/Sequoia/).
 
```

Comment:
This here makes sense. the one at the start doesn't, I think. That's introduced too early